### DISCUSSION TITLE
Next 13 static directory

### DISCUSSION THREAD

**Discussion by rdagostino** [vercel/next.js#43612] [Help]
We upgraded to `Next 13` and seeing a `static` directory appearing in the root of the project all of a sudden. All the assets in there are from the `src/images` which are included in the React components. Prior to upgrading this wasn't happening.

Then the following message appears when running `yarn dev`. We have a public directory currently.
```
warn  - The static directory has been deprecated in favor of the public directory. https://nextjs.org/docs/messages/static-dir-deprecated
```

Is there a `next.config.js` option to generate the directory in public? Is it even needed?

**Comment by PaulRBerg** [comment]
Still seeing this warning in Next.js v13.4.4 despite using a `public` folder.

Nothing that we did made it go away.
